application/lib/viewmodel/map_view_model.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:flutter/material.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+class MapViewModel with ChangeNotifier {
+  late GoogleMapController mapController;
+  late final LatLng _center =
+      const LatLng(46.254633422682765, 20.157634687339225);
+
+  void onMapCreated(GoogleMapController controller) {
+    mapController = controller;
+  }
+
+  CameraPosition get initialCameraPosition => CameraPosition(
+        target: _center,
+        zoom: 11.0,
```

Comment:
Azt hívjuk magic numbernek, amikor minden megnevezés nélkül valahová beírunk egy konstanst. Ez azért lehet veszélyes, mert ha több helyen használnánk ugyan ezt, akkor követhetünk el elírást. Ezen kívül nehezen eldönthető, hogy miért pont az van odaírva ami. Ilyenkor érdemes az osztályba felvenni egy privát constants-t neki. Tehát itt a `MapViewModel`-ben egy `double _initialZoomValue = 11.0` értéket